tests/pa/test_provision.py

```diff
@@ -204,6 +205,7 @@ def _provision_test(self,
         ph.protocol_util.osutil = mock_osutil
         ph.protocol_util.get_protocol = MagicMock()
 
+        conf.get_regenerate_ssh_host_key = Mock(return_value=True)
```

Comment:
just a question: why do we make this True here and at line 267? Thanks

Reply:
I set that twice because they're in different test case function.

Reply:
Sorry, I meant to ask why those 2 tests need this set to true. Thanks

Reply:
that's because in the original code. the test mocked "reg_ssh_host_key" method.
in this PR, it's seperated into two steps, and for reach the second step. we need to mock the get_regenerate_ssh_host_key to true.